test/test_parser.py

```diff
@@ -565,8 +565,8 @@ def test_value_name_whitespace():
         assert_equal(result["items"][i]["properties"]["content"][0]["value"], "One\nTwo\nThree")
         assert_equal(result["items"][i]["properties"]["name"][0], "One\nTwo\nThree")
 
-    assert_equal(result["items"][10]["properties"]["content"][0]["value"], "Hello World      one\n      two\n      three\n    ")
-    assert_equal(result["items"][10]["properties"]["name"][0], "Hello World      one\n      two\n      three\n    ")
+    assert_equal(result["items"][10]["properties"]["content"][0]["value"], "Hello World       one\n      two\n      three\n    ")
```

Comment:
This new test clearly violates the expected result in https://github.com/aaronpk/microformats-whitespace-tests/blob/master/tests/11.json

Reply:
thanks for the catch! true, this comes from the newline immediately after `<pre>`. i'll fix.

Reply:
No! there is no new line after the `<pre>` tag see: https://github.com/aaronpk/microformats-whitespace-tests/pull/1

I was asking about the 7 spaces instead of 6 that were added in this PR.

Reply:
yes. i understood. currently in this PR, that extra space comes from_ that newline, and yes, i fully get that it _should be_ ignored. i will fix that.

Reply:
the plot thickens! this doesn't actually come from the newline at all. it comes from the whitespace after `Hello World`, which is _before_ the `<pre>`. we're currently discussing in IRC on https://chat.indieweb.org/microformats/ . cc @aaronpk who will likely provide the authoritative answer.

Reply:
yup! it comes from the new stuff added in this PR which does not collapse those spaces to nothing-ness.